starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,74 @@
+body {
+    font-size: 19px;
+    font-family: sans-serif;
+    font-weight: lighter;
+    
+}
+hr {
+    width: 52%;
+}
+li{
+    font-size: 17px;
+}
+ul {
+    list-style-type: none;}
+
+header {
+        width: 100% ;
+        height: 500px;
+        background-image: url("../images/apple-pie.jpg"); 
+        background-size: cover;
+        }
+
+h1{ color: white;
+    font-size: 60px;
+    font-weight:200;
+    
+    
+    z-index: 1;
+    position: relative;
+    top: 200px;
+    left: 450px;
+
+}
+.intro  {
+    text-align: justify;
+    padding: 50px 350px 0px 350px;
+}
+
+
+.recipe-info img {
+    width: 300px;
```

Comment:
to center an image you can use 

```
img {
display: block; 
margin: auto; 
}
```

then you don't need to used weird margins